frontend/src/metabase/App.tsx

```diff
@@ -74,17 +75,21 @@ const mapStateToProps = (
 });
 
 class ErrorBoundary extends React.Component<{
-  onError: (errorInfo: ErrorInfo) => void;
+  onError: (errorInfo: ErrorInfo & Error) => void;
+  countError: () => void;
 }> {
   componentDidCatch(error: Error, errorInfo: ErrorInfo) {
-    this.props.onError(errorInfo);
+    this.props.onError({ ...errorInfo, ...error });
```

Comment:
This gives me the impression that `errorInfo` would have a lot of keys, but as far as I understand, it just has a single `componentStack` key.

Not sure what you think about deliberating merging `componentStack` into `error` instead of using spread syntax like this?